packages/core/test/invoke.test.ts

```diff
@@ -2256,7 +2277,9 @@ describe('invoke', () => {
     it('behaviors should have reference to the parent', (done) => {
       const pongBehavior: Behavior<EventObject, undefined> = {
         transition: (_, event, { self }) => {
-          if (event.type === 'PING') {
+          const _event = toSCXMLEvent(event);
```

Comment:
note: this kinda acts as a cast but looks like it has some meaning for this test case - I think that I would prefer this to be casted as that look more honest to the reader